gcloud-java-storage/src/main/java/com/google/gcloud/spi/DefaultStorageRpc.java

```diff
@@ -168,6 +169,7 @@ public StorageObject create(StorageObject storageObject, final InputStream conte
           .setPrefix(PREFIX.getString(options))
           .setMaxResults(MAX_RESULTS.getLong(options))
           .setPageToken(PAGE_TOKEN.getString(options))
+          .setFields(FIELDS.getString(options))
```

Comment:
I assume we need it also on the buckets get and list operations, right?
Add it to the IT test?